anujdeep.ipynb

```diff
@@ -1,32 +1,516 @@
+        "id": "vLs0F-a_NHYy",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        },
+        "outputId": "32e7ae19-f259-4646-ad73-c52c6a4c1165"
+      },
+      "cell_type": "code",
+      "source": [
+        "a=np.arange(20)\n",
+        "print(a[a%2!=0])"
+      ],
+      "execution_count": 19,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "[ 1  3  5  7  9 11 13 15 17 19]\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "id": "RMDyUcP1NclY",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        },
+        "outputId": "caaee8c4-c9bc-459a-ce2a-76b129150115"
+      },
+      "cell_type": "code",
+      "source": [
+        "a = np.array([1,2,3,2,3,4,3,4,5,6])\n",
+        "b = np.array([7,2,10,2,7,4,9,4,9,8])\n",
+        "print(np.intersect1d(a,b))"
+      ],
+      "execution_count": 21,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "[2 4]\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "id": "bIj1tqBfOD2w",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 52
+        },
+        "outputId": "9c17d2b3-c37e-4376-aa68-6c66a1c82cd1"
+      },
+      "cell_type": "code",
+      "source": [
+        "a=np.arange(10).reshape(2,5)\n",
+        "print(a)"
+      ],
+      "execution_count": 23,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "[[0 1 2 3 4]\n",
+            " [5 6 7 8 9]]\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "id": "tzIqo7cdPIw1",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 52
+        },
+        "outputId": "9183a01a-b032-43b7-9913-01fc9e00a268"
+      },
+      "cell_type": "code",
+      "source": [
+        "a = [1, 2, 3, 4, 5, 6, 7, 8, 9]\n",
+        "a=np.array(a)\n",
+        "print(a)\n",
+        "a=list(a)\n",
+        "print(a)"
+      ],
+      "execution_count": 25,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "[1 2 3 4 5 6 7 8 9]\n",
+            "[1, 2, 3, 4, 5, 6, 7, 8, 9]\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "id": "mMUXSLyKPd5Y",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 190
+        },
+        "outputId": "de3ef6a5-debb-4b1a-dd27-f1ee45615d88"
+      },
+      "cell_type": "code",
+      "source": [
+        "b=np.zeros((10,10))\n",
```

Comment:
You were supposed to create a zero array with 1s bordering the zeros. Give it a check.